crypto/bio/bio_lib.c

```diff
@@ -337,7 +337,7 @@ static int bio_write_intern(BIO *b, const void *data, size_t dlen,
 
     if (!b->init) {
         BIOerr(BIO_F_BIO_WRITE_INTERN, BIO_R_UNINITIALIZED);
-        return -2;
+        return -2;               /* shouldn't this be -1? */
```

Comment:
Do you intend to still do anything about this line in this pr? If you just want to leave the comment, maybe you should prefix it with a TODO?   `/* TODO: shouldn't this be -1? */`

Reply:
It's meant as an open question, which I would like to have an answer on here.

In *my* opinion, this should be changed to -1, and in the common case, callers check if the return is `<= 0` or `< 0`, so this should be a safe change.